inc/class-cachify.php

```diff
@@ -808,14 +775,33 @@ public static function flush_notice() {
 	 * @change  2.1.2
 	 *
 	 * @param   integer $id  Comment ID.
+	 *
+	 * @deprecated 2.4 Use comment_edit($id, $comment) instead.
 	 */
 	public static function edit_comment( $id ) {
-		if ( self::$options['reset_on_comment'] ) {
-			self::flush_total_cache();
-		} else {
-			self::remove_page_cache_by_post_id(
-				get_comment( $id )->comment_post_ID
-			);
+		self::comment_edit( $id, array( 'comment_approved' => 1 ) );
+	}
+
+	/**
+	 * Remove page from cache or flush on comment edit.
+	 *
+	 * @since 2.4 Replacement for edit_comment($id) with additional comment parameter.
+	 *
+	 * @param integer $id      Comment ID.
+	 * @param array   $comment Comment data.
+	 */
+	public static function comment_edit( $id, $comment ) {
+		$approved = (int) $comment['comment_approved'];
```

Comment:
Couldn’t we just fetch the comment data with `get_comment()` instead of using the data param added in 4.6, so we do not need to increase the min WP version for that?

Reply:
Yes, we could. In the meantime we raised the requirements to 4.7 anyway, so I guess that's no longer a point here.